### ISSUE TITLE
How much CVE do you have?

### ISSUE THREAD

**Issue by anyela-sweden** [curvance-web/app-community#36]
could be good to have the amount of CVE you have on your wallet. just behind the input (exactly as a swap, you have the amount of both asset just under the inputs.)

<img width="590" alt="Screenshot 311" src="https://github.com/user-attachments/assets/b676c9e1-afc2-4899-8bdc-acfe51f93aaf">

<img width="1007" alt="Screenshot 310" src="https://github.com/user-attachments/assets/496a56a4-38c5-4401-afa8-f8bcb0c5f2d0">

The only way to know how much i have is to press the button MAX. Not convenient.

**Comment by ZeroXMai** [comment]
I like this suggestion, might also be worthwhile having veCVE balance displayed in the app. Gives users clarity on current balances especially with the ability to move CVE and veCVE across chains.

**Comment by joaom00** [comment]
hey, the amount of CVE you have appears right above

![image](https://github.com/user-attachments/assets/9a18ca2b-6914-49f0-bd27-2cc65848939c)

**Comment by anyela-sweden** [comment]
> hey, the amount of CVE you have appears right above
> 
> ![image](https://private-user-images.githubusercontent.com/48808846/361865074-9a18ca2b-6914-49f0-bd27-2cc65848939c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4NzY0NzMsIm5iZiI6MTcyNDg3NjE3MywicGF0aCI6Ii80ODgwODg0Ni8zNjE4NjUwNzQtOWExOGNhMmItNjkxNC00OWYwLWJkMjctMmNjNjU4NDg5MzljLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI4VDIwMTYxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFiMWE1M2M3MzU4MTY1MzIyNTI2Y2ZkNDYxNzllMjkwMGVhMGI1YTc4ZWI5NWNiZWY1MTI1MjI3N2YyOGIxOGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hGtUAl9-tPlCkT25hrRDrWXUimIi4ihRJBocGGTGbIk)

Yep, but while you have to enter the amount to lock, it could be good to avoid a scroll up to check your balance (an a scroll down back to the input) and have it straight to the bottom of the Input as all crypto app actually do while a swap for example. It's a confort UX feature. make sense?
